src/modules/universal-discovery/components/finder/finder.tree.branch.component.js

```diff
@@ -92,9 +102,7 @@ export default class FinderTreeBranchComponent extends Component {
         }
 
         return (
-            <button
-                className="c-finder-tree-branch__load-more"
-                onClick={() => this.props.onLoadMore(this.props.parentLocation)}>
+            <button className="c-finder-tree-branch__load-more" onClick={() => this.props.onLoadMore(this.props.parentLocation)}>
```

Comment:
Please, create a class method for handling `onClick` it wil prevent additional re-renderings on each update.